### DISCUSSION TITLE
Use Iphone/Ipad as a showcase device

### DISCUSSION THREAD

**Discussion by LucianCrainic** [flet-dev/flet#1433] [Q&A] [answered]
Hello,

i was wondering if it is possible to use a device like an Ipad to showcase a Flet app. I saw that there was a Iphone.md file but it was deleted in a recent commit so i am a bit confused about if it's possible or not.

My goal is to host my app on an Ipad that people can use to interact with the app that i built.

**Comment by FeodorFitsner** [comment]
I'm currently working on iOS support. Here is some sneak peek:

https://github.com/flet-dev/flet/assets/5041459/e80c8a5e-d1b7-47bb-8cb7-e40b456f0286

**Comment by LucianCrainic** [comment]
Oh this looks lovely and what i am looking for, is there any way to contribute to it or get access to the development/testing version ?

**Reply by FeodorFitsner** [reply, marked as answer]
Sure, will be publishing docs next week.
